doc/aria-supported.md

```diff
@@ -14,8 +14,7 @@ For a detailed description about how accessibility support is decided, see [How
 
 ## Attributes
 
-| aria-attribute       | axe-core support |
-| -------------------- | ---------------- |
-| aria-describedat     | No               |
-| aria-details         | No               |
-| aria-roledescription | No               |
```

Comment:
We should do something to put the exceptions in here, something like "Everything except button, input[type=button], ...". Even better is we do something like "No [1]" and than under the table we put that list of exceptions as `1. Everything except ...`.